### DISCUSSION TITLE
Several beatmaps missing after import

### DISCUSSION THREAD

**Discussion by gabrieldabbah** [ppy/osu#17522] [Q&A]
Multiple beatmaps are missing (most of them) after importing from a stable installation. I checked the songs folder (From the stable installation), and the missing beatmaps are there. For some reason, they don't show up on lazer. Because there are thousands of beatmaps missing, getting them one by one would be a hassle.

**Comment by peppy** [comment]
https://github.com/ppy/osu/discussions/15188

Make sure to do the above immediately after performing an import.

Also list the beatmaps which you expect to be there but are missing.

**Reply by gabrieldabbah** [reply]
logs
[database.log](https://github.com/ppy/osu/files/8369543/database.log)
[network.log](https://github.com/ppy/osu/files/8369544/network.log)
[performance.log](https://github.com/ppy/osu/files/8369545/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8369546/runtime.log)

Expected beatmaps:

Painters of the tempest
The sun the moon the stars
Rolling pandemonium
thousands others

**Reply by gabrieldabbah** [reply]
the database.log indicates that there are no .osu inside the beatmap folder of the mentioned beatmaps. However, they do have a .osu inside.
![Screen Shot 2022-03-29 at 05 11 13](https://user-images.githubusercontent.com/63445917/160577078-1563f433-9705-44da-b506-f81798a4e62e.png)
 of them.

**Reply by peppy** [reply]
I tested that beatmap and it weirdly works fine for me:

![osu! 2022-04-01 at 04 00 05](https://user-images.githubusercontent.com/191335/161192528-fa88fc16-de1b-4354-a33a-85a1f9385525.png)

I've gone through the code paths for detecting the `.osu` file in a beatmap directory and can't see any immediate issues. Do you have stable installed to `~/.osu/`? Are you able to zip and attach that whole beatmap folder in the screenshot above? There might be something different about the files you have in there compared to a fresh download.
